tests/helpers/linux/test_net.py

```diff
@@ -70,6 +72,17 @@ def test_get_net_ns_by_fd(self):
                 file.fileno(),
             )
 
+    def test_get_socket_by_inode(self):
+        with create_socket(type=socket.SOCK_DGRAM) as skt:
+            inode = fget(self.task, skt.fileno()).f_inode
+            sock = get_socket_by_inode(inode)
+            self.assertEqual(sock.type, socket.SOCK_DGRAM)
+
+        stdin = fget(self.task, sys.stdin.fileno())
```

Comment:
Just in case someone runs the test cases with stdin redirected to a socket (although I see no reason to do that), could you have this test case open `/dev/null` and use that?